### DISCUSSION TITLE
Please put gnome-pie in the Nix Package Manager database if you can

### DISCUSSION THREAD

**Discussion by WMan22** [Schneegans/Fly-Pie#297] [Ideas & Feature Discussions]
I'm sorry if this is the wrong place to ask because it's for Fly-Pie and I don't mean to be a nuisance, but I've been getting more into non arch/immutable distros that don't use GNOME and I basically can't be comfortable without software like this anymore, and I don't know where else to request this since the issue tracker for Gnome-Pie is archived. 

I'm not asking for any kind of update to gnome-pie, it works fine for me as is, but if I'm using, say, my Steam Deck, I can't use this the way I want to because Fly-Pie is GNOME exclusive while gnome-pie is seemingly desktop environment agnostic. Nix Package Manager can be installed in a container, so it'd be deployable on any distro, so I'm humbly requesting in the most desperate, pleading way possible that you add it to the [Nix Package Database,](https://search.nixos.org/packages) as you are the one who made it originally.

As I'm not asking for any actual updates to gnome-pie, I would hope this request falls under the purview of "very basic support", cause it'd suck if that's not in the cards.

**Comment by Schneegans** [comment]
Thank you very much for this kind request! Yet I fear that I won't have the time to look into this. Especially testing the package will require a lot of time as I have never worked with NixOS before. However, I really encourage you to add the package yourself! Gnome-Pie is packaged for many distributions all all packages have been created (and are maintained) by volunteers :smile:

**Reply by WMan22** [reply]
In the time between making this thread and now [I read that you're prototyping a platform agnostic version of Fly-Pie](https://github.com/Schneegans/Fly-Pie/issues/22#issuecomment-1520667903) which would honestly be just as good as if gnome-pie made it to the nix package database, in my opinion. 

The whole reason I wanted gnome-pie in the Nix Package Database is so that I can use it on any distro with any desktop environment (or at the very least KDE Plasma in addition to GNOME). I just want to be able to use Fly-Pie _or_ gnome-pie outside of gnome and on immutable distros like SteamOS, etc. and if this platform agnostic version that's being prototyped gets it working in that manner, then that'd be just as good in my opinion. A flatpak would be best but I'm not sure what kind of permission management issues flatpak would run into for something like gnome/fly-pie.

I would put it in the nix database myself as you suggested, but A: I'm not a programmer and thus am not familiar with how to dance around any kind of code licensing issues to do so, and B: I would be unable to maintain even "very basic support" like security patches and such.
